charts/consent/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: consent
-version: 0.7.0
+version: 0.7.1
```

Comment:
You don't need to bump the chart version numbers. This will be handled automatically when you merge the pr.

Reply:
The action seems to not like it when these aren't bumped at the PR stage, leaving in for now and expecting that the final version will be different.